vumi_twilio_api/twilio_api.py

```diff
@@ -62,10 +79,28 @@ def format_xml(dct, root=None):
 
     @staticmethod
     def format_json(dct):
-        return json.dumps(dct)
+        if dct.get('Call'):
+            dct = dct['Call']
+        if dct.get('Version'):
+            dct = dct['Version']
+        c2s = re.compile('(?!^)([A-Z+])')
+
+        def camel_to_snake(string):
+            return c2s.sub(r'_\1', string).lower()
+
+        def convert_dict_keys(dct):
+            res = {}
+            for key, value in dct.iteritems():
+                if isinstance(value, dict):
+                    res[camel_to_snake(key)] = convert_dict_keys(value)
+                else:
+                    res[camel_to_snake(key)] = value
+            return res
```

Comment:
Could we make `convert_dict_keys` and `camel_to_snake` helper methods (either classmethods or just standalone functions) and make `c2s` a constant defined outside the function?